src/sagemaker/workflow/fail_step.py

```diff
@@ -36,14 +36,15 @@ def __init__(
         """Constructs a FailStep.
 
         Args:
-            name (str): The name of the Fail step.
-            error_message (str or PipelineNonPrimitiveInputTypes): Error message defined by user.
-                Once the Fail step is reached the execution will fail and the
+            name (str): The name of the fail step. A name is required and must be unique within an account.
```

Comment:
> The name of the fail step

I'm thinking about whether we should make `fail` capitalized as `Fail` since it's a step type, and I find that it's inconsistent among existing steps. For example the Training step uses a lower case [here](https://github.com/aws/sagemaker-python-sdk/blob/dev/src/sagemaker/workflow/steps.py#L242) while the CreateModelStep is capitalized [here](https://github.com/aws/sagemaker-python-sdk/blob/dev/src/sagemaker/workflow/steps.py#L337)

> A name is required and must be unique within an account

Should be "A name is required and must be unique within a pipeline". Sorry i didn't make it clear.

Reply:
Thanks for the clarification! I think in that case it's best to go with either `FailStep` or fail step. I can go through and standardize this as well.

Reply:
I would recommend `FailStep`